pkg/core/localize/locales/en/cmd/serviceaccount.en.toml

```
@@ -43,6 +43,9 @@ one = 'internal server error'
 one = 'you are forbidden to {{.Operation}} this service account'
 other = 'you are forbidden to {{.Operation}} these service accounts'
 
+[serviceAccount.common.error.limitExceeded]
+one = 'account can not have more than 50 service accounts'
```

We cannot put any backend driven business logic into message. 
50 account limit is just config on backend that can change etc.

```suggestion
one = 'account can not have more than 50 service accounts'
```